kahuna/public/js/services/telemetry.ts

```diff
@@ -0,0 +1,39 @@
+import {UserTelemetryEventSender, IUserTelemetryEvent} from '@guardian/user-telemetry-client';
+import { v4 } from 'uuid';
+
+const getStoredId = (storage: Storage, key: string): string => {
+    const maybeId = storage.getItem(key);
+    if (maybeId) {
+        return maybeId;
+    } else {
+        const id = v4();
+        storage.setItem(key, id.toString());
+        return id;
+    }
+};
+
+const getBrowserId = () => getStoredId(localStorage, 'browserUuid');
+
+const getSessionId = () => getStoredId(localStorage, 'sessionUuid');
```

Comment:
should this be set on sessionStorage?

Reply:
Well caught! Yes, should be `sessionStorage`. Fixed now!